### PR TITLE
Configuration draft for Bardarash, Kurdistan.

### PR DESCRIPTION
**Pull request by yohanboniface**

### CODE REVIEW

File: ideascube/conf/idb_kur_bardarash.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+"""Bardarash in Kurdistan"""
+from .azraq import *  # noqa
+from django.utils.translation import ugettext_lazy as _
+
+IDEASCUBE_NAME = u"Bardarash"
+USER_FORM_FIELDS = (
```
Comment (yohanboniface) [inline ideascube/conf/idb_kur_bardarash.py]: I think we can keep Azraq ones until we have better knowledge of wanted fields.

File: ideascube/conf/idb_kur_bardarash.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    {
+        'id': 'khanacademy',
+    },
+    {
+        'id': 'wikipedia',
+        'languages': ['en','ar']
+    },
+    {
+        'id': 'gutenberg',
+        'languages': ['en','ar']
```
Comment (yohanboniface) [inline ideascube/conf/idb_kur_bardarash.py]: `lang: mul`

**Comment by barbayellow** [comment]
I didn't even know we could add or remove fields in the user handler app. Can we add any kind of field (image, blob, etc.) ? Any number of fields ? Ore are there any restrictions ?

**Comment by yohanboniface** [comment]
> Can we add any kind of field (image, blob, etc.) ? Any number of fields ? Ore are there any restrictions ?

No strong restriction. The trick is that every field is added for every box, but then each box decides to show only some of them.

**Pull request merged**